OpenRA.Mods.Common/Traits/Air/AttackPlane.cs

```diff
@@ -37,7 +37,7 @@ public override Activity GetAttackActivity(Actor self, Target newTarget, bool al
 			return new FlyAttack(self, newTarget);
 		}
 
-		protected override bool CanAttack(Actor self, Target target)
+		public override bool CanAttack(Actor self, Target target)
```

Comment:
Removed it again. I was foreseeing someone will make me as the reviewers have a tendency to "oh now you touched that legacy code slightly how about redoing it completely" and that is more painful to do afterwards.